resources/migrations/001_update_migrations.yaml

```diff
@@ -9383,10 +9383,69 @@ databaseChangeLog:
                   name: notification_id
             indexName: idx_notification_subscription_notification_id
 
+  - changeSet:
+      id: v51.2024-09-05T08:00:03
+      author: qnkhuat
+      comment: Create the channel_template table
+      preConditions:
+        - not:
+          - tableExists:
+              tableName: channel_template
+      changes:
+        - createTable:
+            tableName: channel_template
+            remarks: custom template for the channel
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(64)
+                  remarks: the name of the template
+                  constraints:
+                    nullable: false
+              - column:
+                  name: channel_type
+                  type: varchar(64)
+                  remarks: the channel type of the template
+                  constraints:
+                    nullable: false
+              - column:
+                  name: details
+                  type: ${text.type}
+                  remarks: the details of the template
```

Comment:
Is this like a description? I'm not quite clear on the purpose from this remark, and it would be good to flesh it out a bit more.

Reply:
it's like metabase_database.details.

The schema of this is different for each template and will be defined by the template.

For example, for email channels, this will store the mustache template